internal/pkg/template/input.go

```diff
@@ -72,6 +77,14 @@ func validateType(fl goValidator.FieldLevel) bool {
 	return fl.Parent().FieldByName("Kind").String() == "input"
 }
 
+func validateRules(fl goValidator.FieldLevel) bool {
+	if fl.Field().IsZero() {
+		return true
+	}
+	_, panicErr := checkPanic(func() error { return validateAgainstRulesString("string", fl.Field().String(), nil) })
```

Comment:
Posílám tam `"string"` což nemusí být validní hodnota, ale to je jedno. Jde o to jestli z toho vypadne panic nebo ne.